hal/network/ncp_client/sara/sara_ncp_client.cpp

```diff
@@ -1092,6 +1097,17 @@ int SaraNcpClient::selectNetworkProf(ModemState& state) {
         CHECK(checkNetConfForImsi());
     }
 
+    // AT+CEMODE? query can take up to 24 seconds to poll without CellularFunctionality::MINIMUM first, and we don't
```

Comment:
Have we changed the decision on only applying this change during intervention?

Reply:
As discussed this morning, yes we were intending on applying this change at power up for the fastest application of CEMODE... however it would also be nice not to have to apply this forcibly after an OTA update to new firmware, so we'll look at moving the code to a "cold boot" location, instead of the current "warm boot" location.

Reply:
As it turns out, this is not a reliable way of applying PS Only mode, so we must make the one time change and deal with the minor disconnect/reconnect.